src/Model_TEST.cc

```diff
@@ -363,3 +365,115 @@ TEST(DOMModel, ToElement)
   for (uint64_t i = 0; i < model2.ModelCount(); ++i)
     EXPECT_NE(nullptr, model2.ModelByIndex(i));
 }
+
+/////////////////////////////////////////////////
+TEST(DOMModel, Uri)
+{
+  sdf::Model model;
+  std::string name = "my-model";
+  ignition::math::Pose3d pose(1, 2, 3, 0.1, 0.2, 0.3);
+  std::string uri =
+    "https://fuel.ignitionrobotics.org/1.0/openrobotics/models/my-model";
+
+  model.SetName(name);
+  model.SetRawPose(pose);
+  model.SetStatic(true);
+  model.SetPlacementFrameName("link0");
+  model.SetPoseRelativeTo("other");
+  model.SetUri(uri);
+  EXPECT_EQ(uri, model.Uri());
+
+  // ToElement using the URI, which should result in an <include>
+  {
+    sdf::ElementPtr elem = model.ToElement();
+    EXPECT_EQ("include", elem->GetName());
+
+    sdf::ElementPtr uriElem = elem->GetElement("uri");
```

Comment:
`GetElement` is unlikely to return a `nullptr` since it creates the element if it's missing. Consider using `FindElement` instead. Here and below.

Reply:
Done in 1eb2d0e